Sources/MessagingPush/MessagingPushImplementation.swift

```diff
@@ -113,6 +133,31 @@ internal class MessagingPushImplementation: MessagingPushInstance {
                                     data: MetricRequest(deliveryId: deliveryID, event: event, deviceToken: deviceToken,
                                                         timestamp: Date()))
     }
+    func getDefaultDeviceAttributes(completionHandler: @escaping([String: Any]?) -> Void) {
```

Comment:
Do you think mocking will work in this case ? Since default attributes are device specific, os specific, sdk specific, by adding mock attributes we would be kind of hard coding the values and tests may fail at times.